### PR TITLE
Update Purge.php

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: Purge.php [hunk 1]
```diff
@@ -1 +1,54 @@
...
+}
+
+$wgExtensionCredits['other'][] = array(
+    'path' => __FILE__,
+    'name' => 'Purge',
+    'author' => 'Tom Hutchison', 
+    'url' => 'https://github.com/Hutchy68/Purge', 
+    'descriptionmsg' => 'descriptionmsg',
+    'version'  => '1.0.0',
+    'license-name' => "License",
```
Comment (kghbln) [inline Purge.php]: @paladox You could add GPL-2.0+ here. The file containing the license is already there.

Comment (paladox) [inline Purge.php]: Ok done.

**Comment by kghbln** [comment]
@Hutchy68 Heiya Tom, ready to merge. :)

**Comment by Hutchy68** [comment]
Thanks @paladox and @kghbln

**Comment by paladox** [comment]
Your welcome.

**Pull request merged**